tests/CMakeLists.txt

```diff
@@ -185,7 +188,7 @@ IF (NOT DISABLE_TESTS)
 				   ${OPENSHOT_TEST_FILES} )
 
 	# Link libraries to the new executable
-	target_link_libraries(openshot-test openshot ${UNITTEST++_LIBRARY})
+	target_link_libraries(openshot-test openshot ${UNITTEST++_LIBRARY} libeffectsuperblur)
```

Comment:
Hmm. If your unit tests are going to be testing the dynamic loading capability for effect plugins (and they probably should), I don't think you want to link the test target with the plugin target.

You may even want to break these up into two separate tests: an unlinked one that just tests libopenshot's dynamic loading features, then another that's linked directly with the plugin itself, and tests its internal APIs.

Reply:
(Note that the above would require multiple separate test executables, which is something I've experimented with moving to anyway for various reasons. See #310 for the WIP stab at that.)

Reply:
Did it switchable. What do you think?